File: frontend/src/app/interfaces/order.ts
import { OrderStates } from "../enums/orderStates";
import { OrderTypes } from "../enums/orderTypes";
import { Docscan } from "./docscan";

export interface Order {
    /** Номер заявки – идентификатор не редактируемый */
    id: string;    
    /** Дата заявки – не редактируется */
    date: string;
    /** Состояние заявки – результат обработки услуги госорганом, не редактируется */
    state: OrderStates;
    /** Статус заявки – системный статус (в таблице ниже), не редактируется */
    status: string;
    /** СНИЛС */
    snils: string;
    /** ФИО заявителя */
    fio: string;
    /** Контактные данные */
    contactData: string;
    /** Тип заявки – выбор: заявление на смену реквизитов в ПФР, запрос мер поддержки */
    type: OrderTypes;
    /** Тело заявки – сгенерированное в XML(JSON) для отправки на сервис СМЭВ */
    body: string;
    /** Ссылки на документы – при подаче заявлений прикрепляются сканы документов, ограничение одного файла 5 Мб */
    documents: Array<Docscan> | null;
    /** Меры поддержки – тело ответа из госоргана при подаче запроса */
    supportMeasures: string;
}

File: backend/src/SecurityService_Core/Models/DB/Order.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace SecurityService_Core.Models.DB
{
    public class Order : BaseEntity
    {
        public DateTime? Date { get; set; }
        public string? State { get; set; }
        public string? SNILS { get; set; }
        public string? FIO { get; set; }
        public string? ContactData { get; set; }
        public string? Type { get; set; }
        public string? Body { get; set; }
        public string? SupportMeasures { get; set; }
    }
}
